src/java.base/share/classes/javax/crypto/Cipher.java

```diff
@@ -510,15 +512,22 @@ private static Transform getTransform(Service s,
      * requirements of your application.
      *
      * @implNote
-     * The JDK Reference Implementation additionally uses the
-     * {@code jdk.security.provider.preferred}
+     * The JDK Reference Implementation additionally uses
+     * <ul>
+     * <li>the {@code jdk.security.provider.preferred}
      * {@link Security#getProperty(String) Security} property to determine
      * the preferred provider order for the specified algorithm. This
      * may be different than the order of providers returned by
      * {@link Security#getProviders() Security.getProviders()}.
      * See also the Cipher Transformations section of the {@extLink
      * security_guide_jdk_providers JDK Providers} document for information
      * on the transformation defaults used by JDK providers.
+     * </li>
+     * <li>the {@code jdk.crypto.disabledAlgorithms}
```

Comment:
Missing "The JDK Reference Implementation additionally uses".

Reply:
This is one of the list items. "The JDK Reference Implementation additionally uses" applies to the entire list which includes both security properties.

Reply:
Ok. I might suggest rewording the first sentence as: "The JDK Reference Implementation additionally uses the following security properties:" and then the list would be:

`<li>{@code jdk.security.provider.preferred}: determines the preferred provider order for the specified algorithm. ...`

Reply:
Sure~